### PR TITLE
Adding Cautions, fixing urls, standardising layout

### PR DESCRIPTION
**Pull request by leflambeur**

### CODE REVIEW

File: content/docs/api-reference/app-registrations-api/index.md [hunk 1]
```diff
@@ -74,24 +69,40 @@ The client secret ***must*** be taken note of at this point, as it will be redac
 }
 ```
 
+{{< caution >}}
+**Caution:** The expiry date refers to the secret only, any tokens generated with this secret will not automatically become invalid when the secret expires or is rotated. Each token has a TTL of 1 hour by default.
```
Comment (robinbryce) [inline content/docs/api-reference/app-registrations-api/index.md:73]: 1 hour by default gives me the sense that I (as the customer) may be able to change that TTL. Could just remove "by default"

File: content/docs/api-reference/app-registrations-api/index.md [hunk 2]
```diff
@@ -74,24 +69,40 @@ The client secret ***must*** be taken note of at this point, as it will be redac
...
-     --data-urlencode "grant_type=client_credentials" \
-     --data-urlencode "client_id=${CLIENT_ID}" \
-     --data-urlencode "client_secret=${SECRET}") && echo "Authorization: Bearer ${TOKEN}" > app-id.jwt
+curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
+    --data-urlencode "grant_type=client_credentials" \
+    --data-urlencode "client_id=${CLIENT_ID}" \
+    --data-urlencode "client_secret=${SECRET}"
 ```
 
-You can then use this bearer token to interact with other RKVST services.
```
Comment (robinbryce) [inline content/docs/api-reference/app-registrations-api/index.md:92]: Ah good to have the link to the standard. I would appreciate that as a developer

**Review by robinbryce — APPROVED** [review]
All looks good to me. One very minor remark. But happy for this to go as is.

File: content/docs/api-reference/app-registrations-api/index.md (hunk 1 above)
Comment (leflambeur) [reply content/docs/api-reference/app-registrations-api/index.md]: Fixed :)

**Pull request merged**